motor/discovery/aws/ec2_instances.go

```diff
@@ -96,6 +110,10 @@ func (ec2i *Ec2Instances) getInstances(account string, ec2InstancesFilters Ec2In
 					log.Debug().Msgf("filtering by tag %s:%s", k, v)
 				}
 			}
+			if ec2i.FilterOptions.InstanceState == 0 {
+				input.Filters = append(input.Filters, types.Filter{Name: aws.String("instance-state-name"), Values: []string{"running"}})
+				log.Debug().Msg("filtering by running state")
```

Comment:
nit:
```suggestion
				log.Debug().Msg("filtering aws instances by running state")
```

gives more context when reading through logs